R/QC4b.R

```diff
@@ -18,7 +18,7 @@
 #'
 
 
-QC4b <- function(d_metingen, verbose = F) {
+QC4b <- function(d_metingen, ph_veld_naam = "h_5__veld", verbose = F) {
```

Comment:
Ik zou default waarden ph_veldnaam op 'pH_veld' zetten, logischer met e.g. geleidbaarheid. Extra parameter moet ook weer genoemd worden in de documentenatie (zie https://github.com/jspijker/KRWQCprotocol/pull/16#discussion_r730724041)